x-pack/server/lib/check_license/check_license.js

```diff
@@ -6,7 +6,7 @@
 
 import { i18n } from '@kbn/i18n';
 import { RANKED_LICENSE_TYPES } from '../constants';
-import { LICENSE_STATUS } from '../../../common/constants';
+import { UNAVAILABLE, INVALID, EXPIRED, VALID } from '../../../common/license_status';
```

Comment:
personally, I don't like exporting/importing very generic constant names since the context is lost once the code moves past the import line. on this line, it's easy to know that `UNAVAILABLE ` is a type of license status from the imported file name, but the constant usages later in the file are not so clear as to what `UNAVAILABLE` means.

that's why I liked nesting them inside `LICENSE_STATUS` object, it gives these generic constant names a very clear scope. prefixing instead of nesting would also work:
```export const LICENSE_STATUS_UNAVAILABLE = 'UNAVAILABLE';```

what do you think?